streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java

```diff
@@ -311,6 +311,7 @@ public void onRestoreEnd(TopicPartition topicPartition, String storeName, long t
      *
      * @param consumerRecord the record to be processed
      */
+    @SuppressWarnings("deprecation")
```

Comment:
Not clear which callee is deprecated?

Reply:
It's `consumerRecord.checksum()` -- but we can't annotate within the method, can we?

Should I add a comment instead or do you want the annotation be removed?

Reply:
Could we just use `NULL_CHECKSUM`?

Reply:
Well. Yes and no. We actually just pass the checksum from the input `ConsumerRecord` -- thus, we would loose this information. However, I guess nobody should use checksum and the maybe we don't need to pass the checksum necessarily?

Reply:
Yup, if people mistakenly validate checksums that is not encouraged as in the deprecation message, so I think we should be fine.

Reply:
Ack.